docs/designers-developers/developers/slotfills/main-dashboard-button.md

```diff
@@ -0,0 +1,44 @@
+# MainDashboardButton
+
+This slot allows replacing the default main dashboard button that's used for closing
+the editor in fullscreen mode.
+
+## Example
+
+```js
+import { registerPlugin } from '@wordpress/plugins';
+import { __experimentalMainDashboardButton } from '@wordpress/edit-site';
```

Comment:
so this slot is just an edit-site slot? Would we need to offer the same for the other screens. Also, if the fallback button is the same across screens, should the slot be the same across screens too?

Reply:
This shouldn't block this PR. I think it's ready.

Reply:
> so this slot is just an edit-site slot?

Yes.

> Would we need to offer the same for the other screens. Also, if the fallback button is the same across screens, should the slot be the same across screens too?

I was planning to make a separate slot for post editor. The rationale being that someone might want to provide a different component in other screens. Or it might look the same but we might want it to behave differently.

Reply:
I'm not 100% certain we need a separate component especially if we add support to more and more screens (widgets...). Also it seems possible to enqueue the extension based on the screen or check the screen and change the behavior.

Reply:
> Also it seems possible to enqueue the extension based on the screen or check the screen and change the behavior.

That's true. Since I was planning on starting a separate PR for post editor, would it be fine if I consolidate this to use the same slot in it?

Reply:
Yes, we should merge this and see in a separate PR if we can have the same slot.